common/next/next.config.js

```diff
@@ -75,11 +78,43 @@ const createConfig =
         return config;
       },
       eslint: {
+        ...validDefaultConfig.eslint,
         ignoreDuringBuilds: !options.lintBuilds,
       },
       transpilePackages: ['@weco/common'],
+      experimental: {
+        ...validDefaultConfig.experimental,
+        mdxRs: true,
+        outputFileTracingRoot: path.join(__dirname, '../../'),
+      },
+      reactStrictMode: true,
     });
     return nextConfig;
   };
 
+// TODO: check the build output without this function whenever next has an update
+/**
+ * this will only be neccessary until the default config is updated OR the config validator is updated.
```

Comment:
typo `necessary`, and if we can link the issue (https://github.com/cyrilwanner/next-compose-plugins/issues/59) this would help us when we check the TODOs once in a while ( 👀 )

Reply:
done